eng/testing/linker/trimmingTests.targets

```diff
@@ -68,6 +69,16 @@
       <_additionalProjectSourceFiles Include="%(TestConsoleApps.AdditionalSourceFiles)" />
     </ItemGroup>
 
+    <ItemGroup>
+      <_switchesAsItems Include="%(TestConsoleApps.DisabledFeatureSwitches)" Value="false" />
+      <_switchesAsItems Include="%(TestConsoleApps.EnabledFeatureSwitches)" Value="true" />
+    </ItemGroup>
+
+    <PropertyGroup>
+      <_extraTrimmerArgs>$(_extraTrimmerArgs) @(_switchesAsItems->'--feature %(Identity) %(Value)', ' ')</_extraTrimmerArgs>
```

Comment:
You don't have to do this. Instead, just set `Trim=true` on the `RuntimeHostConfigurationOption` below - like we do in the SDK:

https://github.com/dotnet/sdk/blob/714082d0e96cef8c3a5f5d04a08d18baaf036f21/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L437-L440

Reply:
Ah, right. Thanks - forgot about that.